Fhi.HelseId.Web/Hpr/HprService.cs

```diff
@@ -12,7 +14,7 @@ namespace Fhi.HelseId.Web.Hpr
     public interface IHprService
     {
         Task<bool> SjekkGodkjenning(string hprnummer);
```

Comment:
Just a heads up - only this method is being used on this interface :)

Reply:
Can we discuss this sometime?